rootfs/api/tests/test_build.py

```diff
@@ -114,14 +114,17 @@ def test_build_default_containers(self):
         response = self.client.post(url, json.dumps(body), content_type='application/json',
                                     HTTP_AUTHORIZATION='token {}'.format(self.token))
         self.assertEqual(response.status_code, 201)
-        url = "/v2/apps/{app_id}/containers/cmd".format(**locals())
-        response = self.client.get(url,
-                                   HTTP_AUTHORIZATION='token {}'.format(self.token))
+
+        url = "/v2/apps/{app_id}/pods/cmd".format(**locals())
+        response = self.client.get(url, HTTP_AUTHORIZATION='token {}'.format(self.token))
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.data['results']), 1)
         container = response.data['results'][0]
         self.assertEqual(container['type'], 'cmd')
-        self.assertEqual(container['num'], 1)
+        self.assertEqual(container['release'], 'v2')
+        # pod name is auto generated so use regex
+        self.assertRegex(container['name'], app_id + '-v2-cmd-[a-z0-9]{5}')
```

Comment:
Yeah, I'll do it after this PR tho since I have a few test additions i want to do that are not directly affecting this PR